pom.xml

```diff
@@ -28,9 +28,26 @@
     <module>jlibra-core</module>
     <module>jlibra-examples</module>
     <module>jlibra-grpc</module>
+    <module>jlibra-integration-tests</module>
   </modules>
 
-  <build>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-failsafe-plugin</artifactId>
+                    <version>3.0.0-M3</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <version>3.0.0-M3</version>
```

Comment:
Couldn't we take a Release or GA or something here?

Reply:
You're right. Changed to 2.22.2

Reply:
I was sure I pushed my commit but I guess I haven't yet. Not sure what happened

Reply:
fixed the version numbers now

Reply:
Looks like 3.0.0-M3 has to stay. I tried 2.22.2 but it crashes the build on circleci.